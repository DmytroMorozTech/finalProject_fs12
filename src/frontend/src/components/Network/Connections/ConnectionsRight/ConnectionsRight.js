import styles from './styles'
import ImgToJobsPage from '../../../../shared/ImgToJobsPage/ImgToJobsPage'
import React from 'react'

function ConnectionsRight () {
  const classes = styles()

  return (
    <div className={classes.connectionsRight}>
      <ImgToJobsPage/>
    </div>
  )
}

export default ConnectionsRight
