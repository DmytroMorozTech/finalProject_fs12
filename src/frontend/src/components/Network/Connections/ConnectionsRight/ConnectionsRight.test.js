import React from 'react'
import ConnectionsRight from './ConnectionsRight'
import { render } from 'enzyme'

const setUp = () => render(<ConnectionsRight />)

describe('should render ConnectionsRight component', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
