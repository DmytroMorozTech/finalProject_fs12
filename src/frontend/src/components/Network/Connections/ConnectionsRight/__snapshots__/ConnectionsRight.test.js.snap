// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ConnectionsRight component to match snapshot 1`] = `undefined`;
